pages/forms/hxh/index.html

```diff
@@ -3,19 +3,20 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles.css">
     <title>We Life</title>
   </head>
   <body>
     <header>
-      <h1>Hunter X Hunter</h1>
+      <title>Hunter X Hunter</title>
```

Comment:
It seems like some changes were overwritten while resolving conflicts. Can you please check?